rcon/game_logs.py

```diff
@@ -120,7 +120,7 @@ def on_connected(insert_at: int | None = None):
     """Insert the given hook at `insert_at` position, or the end"""
 
     def wrapper(func):
-        if insert_at and isinstance(insert_at, int):
+        if insert_at:
```

Comment:
I think `isinstance(insert_at, int)` is better, for the same reasons you pointed out. Sorry when my first comment wasn't clear enough :) I wanted to modify this statement to:

```
if isinstance(insert_at, int):
```
as this will check for None as well.

Reply:
I am bad at reading!